src/dotnet/Program.cs

```diff
@@ -111,7 +111,7 @@ internal static int ProcessArgs(string[] args, ITelemetry telemetryClient = null
                         HelpCommand.PrintHelp();
                         return 0;
                     }
-                    else if (args[lastArg].StartsWith("-"))
+                    else if (args[lastArg].StartsWith("-", StringComparison.OrdinalIgnoreCase))
```

Comment:
Is new parser checking as this as well?  Ideally we should replace parsing in program.cs with the new parser... and I will do it someday